lib/src/Pos/Util/BackupPhrase.hs

```diff
@@ -23,6 +23,7 @@ import           Pos.Crypto (AbstractHash, EncryptedSecretKey, PassPhrase, Secre
                              unsafeAbstractHash)
 import           Pos.Util.Mnemonics (fromMnemonic, toMnemonic)
 import           Test.QuickCheck (Arbitrary (..), genericShrink)
+import           Test.QuickCheck.Instances ()
```

Comment:
I see no other changes in this file. Why did you decide to import this?

Reply:
because it didn't find `Arbitrary Text` instance when I had merged w/ master. dunno why, gromak give advice to me to add this import and it helped